hcipy/optics/detector.py

```diff
@@ -130,6 +145,8 @@ def __init__(self, input_grid, dark_current_rate=0, read_noise=0, flat_field=0,
 		self.read_noise = read_noise
 		self.flat_field = flat_field
 		self.include_photon_noise = include_photon_noise
+		self.subsampling = subsampling
+		self.statistic = statistic
```

Comment:
You should make a distinction between `input_grid` (the grid on which the incoming wavefront should be defined) and `detector_grid` (the grid of the camera image). The input grid is now used as the detector grid. Not sure which one to use as the argument for `__init__()`. If input grid is used, you should check whether subsampling can be performed on this grid (in the initializer to catch mistakes early). Using detector grid as the initializer seems the right choice to me, but you should look at how that works in practice for a few minutes to make sure this is actually the better choice.